app/models/models.go

```diff
@@ -10,29 +10,28 @@ import (
 type MessageStructure string
 type Protocol string
 
-// TODO - reconcile this with app.SqsMessageAttributeValue - deal with ConvertToOldMessageAttributeValueStructure
-type MessageAttributeValue struct {
-	BinaryListValues []string `json:"BinaryListValues,omitempty"` // currently unsupported by AWS
-	BinaryValue      string   `json:"BinaryValue,omitempty"`
-	DataType         string   `json:"DataType,omitempty"`
-	StringListValues []string `json:"StringListValues,omitempty"` // currently unsupported by AWS
-	StringValue      string   `json:"StringValue,omitempty"`
+type MessageAttribute struct {
+	BinaryListValues []string `json:"BinaryListValues,omitempty" xml:"BinaryListValues,omitempty"` // currently unsupported by AWS
+	BinaryValue      []byte   `json:"BinaryValue,omitempty" xml:"BinaryValue,omitempty"`
```

Comment:
@kojisaikiAtSony So, this PR is some significant work toward normalizing the MessageAttributes as we previously discussed.  I think this piece specifically will solve your issue in https://github.com/Admiral-Piett/goaws/pull/330.  There's a lot of other overhaul here though I know.

What do you think?  I can build it for you first if you'd like to test it before hand and make sure it works for your use cases.  Let me know!

Reply:
Great! We'll check this work is able to be alternative with #330!